src/app/components/get-this-title/get-this-title.scss

```diff
@@ -0,0 +1,17 @@
+@import 'variables';
+
+$darker-cyan: #0097a7;
+
+.get-this-title {
+    background-color: white;
+
+    .option {
+        color: color(cyan);
+        cursor: pointer;
+        width: 25rem;
+
+        .fa {
+            color: $darker-cyan;
```

Comment:
Try to use the global variables.  If you need to slightly darken a global, use SCSS's built-in `darken()` function.  For example: `color: darken(color(cyan), 5%);`